modules/ROOT/pages/activity-tracking.adoc

```diff
@@ -0,0 +1,25 @@
+= Activity Tracking
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
```

Comment:
I don't understand the relationship of "activity" "message" and "transactions" Is a transmission different from a message flow?

Reply:
Yes it is. But if you are having trouble with these defs, then more explanation is needed.